src/components/buttons/SecondaryBtn.tsx

```diff
@@ -8,22 +8,24 @@ export const SecondaryBtn = ({
   buttonContent,
   handleClick,
   enableBackground,
+  noBackground,
 }: {
   startIcon?: ReactNode
   buttonContent: ReactNode
   handleClick?: () => void
   enableBackground?: boolean
+  noBackground?: boolean
 }) => {
   return (
     <Button
       variant="outlined"
       startIcon={startIcon ? startIcon : null}
       sx={(theme) => ({
         textTransform: 'none',
-        border: enableBackground ? 'none' : `1px solid ${theme.color.borders.border}`,
+        border: enableBackground || noBackground ? 'none' : `1px solid ${theme.color.borders.border}`,
         bgcolor: enableBackground ? theme.color.gray[150] : '',
         '&:hover': {
-          border: enableBackground ? 'none' : `1px solid ${theme.color.borders.border}`,
+          border: enableBackground || noBackground ? 'none' : `1px solid ${theme.color.borders.border}`,
           bgcolor: theme.color.gray[150],
```

Comment:
This comparison is bit hard to understand because if I don't want background then my props would be `enableBackground={false}` and `noBackground={true}`, which is confusing to implement the behaviour needed.
Can we change the `noBackground` to `outlined`?